go.mod

```diff
@@ -107,7 +107,7 @@ require (
 	github.com/hashicorp/vault-plugin-mock v0.16.1
 	github.com/hashicorp/vault-plugin-secrets-ad v0.10.0
 	github.com/hashicorp/vault-plugin-secrets-alicloud v0.9.0
-	github.com/hashicorp/vault-plugin-secrets-azure v0.10.0
+	github.com/hashicorp/vault-plugin-secrets-azure v0.6.3-0.20210924190759-58a034528e35
```

Comment:
I don't know why this is going to 0.6.3, but the commit hash is correct.

Reply:
I think the repo needs to be tagged to v0.11.0 and we add that here. Thoughts?

Reply:
I was going to wait on tagging the repo since I'm going to be working on root credential rotation next. I can tag it now though if you prefer?

Reply:
Yeah good point, let's wait on tagging.